tests/controller_tests/test_ingest.py

```diff
@@ -411,7 +411,7 @@ def test_ingest_endpoint_returns_uncaught_ingest_error(test_client, example_data
             )
 
     # logs an error
-    assert "ERROR    Sample API:errors.py:32 Uncaught ingest exception." in caplog.text
+    assert "ERROR    Sample API:errors.py:34 Uncaught ingest exception." in caplog.text
```

Comment:
ah we can probably change this to:
```
assert "ERROR" in caplog.text
assert "Uncaught ingest exception." in caplog.text
```
so it isn't dependent to the code location